fuzz/fuzz_targets/fuzz_target_1.rs

```diff
@@ -2,13 +2,42 @@
 
 use libfuzzer_sys::fuzz_target;
 
+
+fn parse(data: &[u8]) -> Vec<Vec<u8>> {
```

Comment:
Why not use [arbitrary](https://github.com/rust-fuzz/arbitrary) ?

Reply:
Since we agreed on not using arbitrary, we can keep that but I have a couple changes I would suggest.